emperor/qiime_backports/parse.py

```diff
@@ -263,8 +258,8 @@ def parse_coords(lines):
     #finally, dump the rest of the lines into a table
     header, result = [], []
     for line in lines[:-2]:
-        fields = map(strip, line.split('\t'))
+        fields = list(map(strip, line.split('\t')))
         header.append(fields[0])
-        result.append(map(float, fields[1:]))
+        result.append(list(map(float, fields[1:])))
```

Comment:
could this block be done as
```python
fields = map(strip, line.split('\t'))
header.append(next(fields))
result.append(lmap(float, fields)))
```
that way the fields list does not have to be created.